pkg/expect/expect_test.go

```diff
@@ -40,3 +41,23 @@ func TestEcho(t *testing.T) {
 		t.Fatalf("expected error on closed expect process")
 	}
 }
+
+func TestSendLinePython(t *testing.T) {
```

Comment:
could this use something from coreutils instead? expect on `/bin/tr a b`, `Sendline("a")`, `Expect("b")`